core/src/main/java/co/aikar/commands/RegisteredCommand.java

```diff
@@ -155,6 +155,7 @@ void handleException(CommandIssuer sender, List<String> args, Exception e) {
         } else {
             sender.sendMessage("&cI'm sorry, but there was an error performing this command.");
             this.scope.manager.log(LogLevel.ERROR, "Exception in command: " + command + " " + ACFUtil.join(args), e);
+            this.scope.manager.handleUncaughtException(sender, args, e);
```

Comment:
This should be a true/false, and if true,don't send the default message (but we can still always print the log)
So plugins can return false to use the default messaging, or send their own and return true.